samples/Commerce/Commerce.UI/App.xaml

```diff
@@ -33,8 +33,9 @@
 			<x:Double x:Key="LandscapeMaxPageWidth">500</x:Double>
 
 			<!-- To override the default FlyoutLightDismissOverlayBackground resource -->
-			<SolidColorBrush x:Key="FlyoutLightDismissOverlayBackground"
-							 Color="{StaticResource OverlayColor}" />
+			<!-- TODO: This override is breaking the flyouts -->
```

Comment:
@carldebilly I'm not sure what happened with the latest style changes but there's a few styles/resources that aren't resolving. Can you take a look at the TODO items in this PR to see what's missing?